runtime/doc/options.txt

```diff
@@ -6280,6 +6280,13 @@ A jump table for the options with a short description can be found at |Q_op|.
 	Also see 'copyindent'.
 	Use |:retab| to clean up white space.
 
+					*'pressenterwait'* *'pew'*
+'pressenterwait'	number	(default 0)
+			global
+	When the value is zero or less, the |press-enter| prompt is enabled.
+	When it is less than zero, it means |:sleep| ms is used to allow the
+	user to verify the output
```

Comment:
In the second sentence, shouldn't it say: When it is **greater** than zero, it means `|:sleep|` ms is used to allow the user to verify the output.

Reply:
Fixed